src/lib/pages/contract-details/components/contract-description/index.tsx

```diff
@@ -11,7 +13,12 @@ interface ContractDescProps {
 export const ContractDesc = ({ contractData }: ContractDescProps) => (
   <Flex gap={6}>
     {contractData.publicProject.publicInfo?.description && (
-      <PublicContractDesc contractData={contractData} />
+      <PublicDescription
+        title="Public Contract Description"
+        description={contractData.publicProject.publicInfo.description}
+        textLine={textLine(!contractData.contractLocalInfo?.description)}
```

Comment:
I think textLine util implementation is a bit weird. I would delete the util fn, set default textLine value to 4 and just ternary when the desc is not present